scipy-1.0/paper.tex

```diff
@@ -567,14 +567,14 @@ \subsection*{SciPy Matures}
 
 By the middle to late 2000s, SciPy was starting to mature after a long phase of significant
 growth and adoption.
-The informal workshops are grown into international conferences with many
-hundreds of attendees, special issues are organized and published in a
-leading scientific magazine, and the scope of the SciPy library
-narrows, while the breadth of the ecosystem is grown and encouraged
+The informal workshops grew into international conferences with many
+hundreds of attendees. Special issues were organized and published in a
+leading scientific journal, and the scope of the SciPy library
+narrowed, while the breadth of the ecosystem grew
 through a new type of auxiliary package, the scikit.  The tooling,
-development, and release processes become more professional, grown with
-the patience affordable in open source development, leading with
-best practices nowadays increasingly common in both industry and other
+development, and release processes became more professional, grown with
+the patience affordable in open source development. Leading with
```

Comment:
I'm not sure if that is what was meant by the original sentence. Consider whether the author meant that SciPy was "grown with patience..." and "lead with best practices...."; "Leading with best practices is increasingly common..." was not a separate thought. I'm not sure that it would be a statement worth including, in any case.

How about:
"The tooling, development, and release processed became more professional. SciPy was expanded carefully, with the patience affordable in open source projects and via best practices common in industry."
(If I understand correctly, these best practices were already common in industry and becoming increasingly common in open source projects. I leave out the bit about them becoming increasingly common in open-source.)
I would consider leaving the last bit about best practices out unless we can give more detail about what these best practices were.

Reply:
Ok, I tried to adopt your approach. I didn't write the original sentence so I can't speak to its intended meaning.